src/main.cpp

```diff
@@ -54,6 +55,12 @@ static void enforce_infinite_rlimit() {
 
 int main(int argc, char *argv[])
 {
+  if (geteuid() != 0)
+  {
+    std::cerr << "ERROR: bpftrace currently only supports running as the root user." << std::endl;
+    exit(0);
```

Comment:
Change exit status code to non-zero to indicate bpftrace exited with error: 

```suggestion
    exit(1);
```